doc/input-output-reference/src/overview/group-electric-load-center-generator.tex

```diff
@@ -1326,7 +1334,7 @@ \subsubsection{Outputs}\label{electricloadcenter-storagebattery-outputs}
 \item
   HVAC,Average, Electric Storage Operating Mode Index {[]}
 \item
-  HVAC,Average,Electric Storage Charge State{[}Ah{]}
+  HVAC,Average,Electric Storage Charge State{[}Ah{]}  
```

Comment:
It looks like there are 2 spaces at the end of this line. CI Clang will catch this if true and can be corrected on merge. No need for another commit to fix this if it is the only change.